meta-mender-core/recipes-mender/mender-client/mender-client-go.inc

```diff
@@ -120,15 +120,11 @@ FILES:${PN} += "\
     ${_MENDER_PERSISTENT_DIR}/mender.conf \
 "
 
-FILES:${PN} += "\
-    /usr/share/dbus-1/interface \
-    /usr/share/dbus-1/interface/io.mender.Update1.xml \
-    /usr/share/dbus-1/interface/io.mender.Authentication1.xml \
-"
 
-
-FILES:${PN}-dev += " \
+FILES:${PN}-dev:append = " \
```

Comment:
I don't know why `append` expands correctly, and `+=` doesn't. It's some deep logic in poky's `renameVar`. It appears to be specific to the `FILES:${PN}-dev` variable.